content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,139 @@
+=======================================================
+Connecting the Windows IoT software to an Odoo database
+=======================================================
+
+A Virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
+computer. This requires a Windows operating system with an Odoo 16 or later database.
+
+The Windows Virtual :abbr:`IoT (Internet of Things)` box works the same way as a physical
+:abbr:`IoT (Internet of Things)` box, with the ability to run most of the same devices. All
+:abbr:`POS (Point of Sale)` devices work with it, such as a scale or printer. Payment terminals will
+also work, but it should be noted that :abbr:`MRP (Material Requirement Planning)` devices are not
+compatible. *These include cameras or measurement tools.*
+
+Pre-requisites
+==============
+
+- Odoo 16 database or any version above
+- :abbr:`IoT (Internet of Things)` compatible devices (except those mentioned above)
+
+  .. seealso::
+     `Odoo's compatible IoT devices <https://www.odoo.com/app/iot-hardware>`_
+
+- Device drivers for Windows
+
+  .. note::
+     Odoo recommends using an updated, recent version of Windows as some older operating systems can
+     cause the Windows virtual :abbr:`IoT (Internet of Things)` to not work.
+
+- Windows computer (laptop, desktop, server)
+- Odoo IoT subscription
+
+  .. seealso::
+     :ref:`iot/iot-eligibility`
+
+Connect the Windows virtual Iot box to an Odoo database
+=======================================================
+
+#. Navigate to the Odoo 16 or higher installation package for Enterprise/Community Windows at
+   `Odoo's Download Page <https://odoo.com/download>`_.
+
+#. Install and setup the Odoo .exe file.
+
+   After the instructions screen click :guilabel:`Next` to start the installation. Agree to the
+   :abbr:`TOS (Terms of Service)`.
+
+#. During the next step of the installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select
+   the type of install` dropdown.
+
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
```

Comment:
I had to add another space for these sub-bullet points to get the sentence after (in the parent bullet) to format correctly.

```suggestion
    - **Odoo server**
    - **Odoo IoT**
    - **Nginx WebServer**
    - **Ghostscript interpreter**
```